packages/@aws-cdk/aws-apigateway/test/test.method.ts

```diff
@@ -693,6 +693,57 @@ export = {
       });
     }, /Authorization type is set to NONE which is different from what is required by the authorizer/);
 
+    test.done();
+  },
+  'method has Auth Scopes'(test: Test) {
```

Comment:
New line between test cases

```suggestion

  'method has Auth Scopes'(test: Test) {
```